wicker/core/column_files.py

```diff
@@ -196,37 +251,38 @@ def __init__(
         :param filelock_timeout_seconds: number of seconds after which to timeout on waiting for downloads,
             defaults to -1 which indicates to wait forever
         :type filelock_timeout_seconds: int, optional
+        :param path_factory: Path factory for determining paths to col files from root
+        :type path_factory: WickerPathFactory, defaults to S3PathFactory
+        :param storage: Storage used for grabbing files. Defaults to nont and creates S3DataStorage if none.
+        :type storage: Optional[AbstractDataStorage]
         :param dataset_name: name of the dataset, defaults to None
         :type dataset_name: str, optional
         """
```

Comment:
Let's make two changes to the private member variables for this class. Let's rename `self._s3_storage` to `self._storage` and let's access `self._columns_root_path` from the `ColumnBytesFileReader` class instead (where we renamed it to `self._column_bytes_root_path`).

These renames should be safe since they are "underscore" private class members - this means that other external users of the Wicker API should not be trying to access them.